test/functional/admin_test.js

```diff
@@ -1,6 +1,6 @@
 Feature('Admin Web');
 
-Scenario('add all the roles', I => {
+Scenario('add all the roles @pipeline', I => {
```

Comment:
any reason we don't want these running nightly?

Reply:
These are for upload config and and roles creation not actual tests -which are already there on AAT.